src/BlazorWasmSdk/Tasks/ComputeBlazorBuildAssets.cs

```diff
@@ -18,6 +18,8 @@ public class ComputeBlazorBuildAssets : Task
         [Required]
         public ITaskItem[] Candidates { get; set; }
 
+        public ITaskItem CustomIcuCandidate { get; set; } = new TaskItem();
```

Comment:
Is there a reason to initialize this property, typically we just check against the default if not present.